.travis/generate_changelog.sh

```diff
@@ -10,8 +10,8 @@ fi
 
 ORGANIZATION=$(echo "$TRAVIS_REPO_SLUG" | awk -F '/' '{print $1}')
 PROJECT=$(echo "$TRAVIS_REPO_SLUG" | awk -F '/' '{print $2}')
-GIT_MAIL="pawel+bot@netdata.cloud"
-GIT_USER="netdatabot"
+GIT_MAIL=${GIT_MAIL:-"pawel+bot@netdata.cloud"}
```

Comment:
να πούμε στον @cakrit 	να φτιάξει ένα `automation@netdata.cloud` ?

Reply:
This email address doesn't matter (it is saved only in git history) so there is no need to create actual inbox. I just used email which was used for @netdatabot account registration.